Description: Replace exit() with abort() in shared library
 The library was calling exit() on error, which terminates the host program.
 This patch replaces it with abort() to allow for proper debugging and crash handling.
Author: Guilherme Eduardo Corrêa Raposo <guilhermeraposo@alunos.utfpr.edu.br>
Forwarded: not-needed
Last-Update: 2025-11-30

--- libvisual-0.4.2.orig/libvisual/lv_error.c
+++ libvisual-0.4.2/libvisual/lv_error.c
@@ -225,7 +225,7 @@ int visual_error_raise ()
 #if !defined(VISUAL_OS_WIN32)
 		raise (SIGTRAP);
 #endif
-		exit (1);
+		abort ();
 	}
 
 	return error_handler (error_handler_priv);
